v2/emailpassword/custom-ui/init/user-management-dashboard/setup.mdx

```diff
@@ -257,15 +257,19 @@ init(
 
 ## Content Security Policy
 
-If you return a `Content-Security-Policy` header in from your backend, you will need to include the following directives for the user management dashboard to work correctly
+If your backend returns a `Content-Security-Policy` header, you will encounter the following UI displaying the CSP violation details. Follow the instructions provided in this UI to make necessary adjustments to your backend CSP configuration.
 
+![CSP error handled UI](/img/dashboard/csp-error.png)
+
+For example, you can resolve the aforementioned error by incorporating the blocked URI domain into your CSP configuration under the violated directive, which, in this case, is `img-src`. Learn more [here](https://content-security-policy.com/)
 ```text
 script-src:
   'self' 
   'unsafe-inline' 
-  https://cdn.jsdelivr.net/gh/supertokens/
```

Comment:
why is this removed from here and added to img-src?

Reply:
Since this example focuses on resolving the violated directive `img-src` only.